src/Components/FormTextArea.js

```diff
@@ -20,22 +27,34 @@ function FormInput({ label, onChange, property, required, showRequiredHint, valu
         onChange={(e) => onChange(property, e.target.value)}
         onFocus={() => setFocused(true)}
         required={required}
-        className="usa-textarea"
+        className={classNames('usa-textarea', {
+          'usa-input--error': validationState === ValidationState.ERROR,
+          'usa-input--success': validationState === ValidationState.FIXED,
+        })}
       />
+      {children}
```

Comment:
I changed the `customHint` prop to `children` for the same reason as FormRadioFieldSet- it's cleaner (and more flexible) to add things in as nested elements rather than in a prop...